src/medCore/process/medAbstractProcess.h

```diff
@@ -32,6 +32,8 @@ class MEDCORE_EXPORT medAbstractProcess : public dtkAbstractProcess
     medAbstractProcess( medAbstractProcess * parent = NULL );
     virtual ~medAbstractProcess();
 
+    using dtkAbstractProcess::setInput;
```

Comment:
Yes I get your point. 

I'm just saying that it's not possible for now (You would be lucky that the implementation of itkThresholdingProcess uses a setInput(medAbstractData *data).

But it will be possible when the process layer will be present in the core of medInria, you will have then a medAbstractThresholdImageProcess or something like that exposing the methods that you need to setup the implementation of your process.